chess/src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import AppCSS from './App.module.css';
+import Board from './components/Board'
+
+function ChessApp() {
+  return (
+    <div className={AppCSS.app} >
```

Comment:
rename to generic classes. everywhere. if the component grows, we would be able to follow the generic naming conventions and therefore it will be easier to you in the future.